ddtrace/internal/agent.py

```diff
@@ -14,6 +14,9 @@
 DEFAULT_STATS_PORT = 8125
 DEFAULT_TRACE_URL = "http://%s:%s" % (DEFAULT_HOSTNAME, DEFAULT_TRACE_PORT)
 DEFAULT_TIMEOUT = 2.0
+DEFAULT_BUFFER_SIZE = 8 * 1000000  # 8mb
```

Comment:
I think these belong in the writer module as they aren't specific to the agent

Reply:
yeah, think I realized that too